build/_build/gitlab.bat

```diff
@@ -0,0 +1,16 @@
+if not exist c:\mnt\ goto nomntdir
+@echo PARAMS %*
+
+call "%VSTUDIO_ROOT%\vc\auxiliary\build\vcvars64.bat"
```

Comment:
The bat file calls _vcvars64.bat_ to set the VS C++ compiler bits, which sets a bunch of environment variables. I couldn't find a nice way of doing that outside of the approach used here (and used previously)